welcome/cloud-experts-rosa-hcp-sts-explained.adoc

```diff
@@ -0,0 +1,116 @@
+
+[id="components-specific-to-rosa-hcp-with-sts"]
+== Components of {hcp-title}
+* *AWS infrastructure* - The infrastructure required for the cluster including the EC2 instances, storage, and networking components. See xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-aws-compute-types_rosa-service-definition[AWS compute types] to see the supported instance types for compute nodes and xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-ec2-instances_rosa-sts-aws-prereqs[provisioned AWS infrastructure] for more information on control plane and infrastructure node configuration.
+* *AWS STS* - A method for granting short-term, dynamic tokens to provide users the necessary permissions to temporarily interact with your AWS account resources.
+* *OpenID Connect (OIDC)* - A mechanism for cluster Operators to authenticate with AWS, assume the cluster roles through a trust policy, and obtain temporary credentials from STS to make the required API calls.
+* *Roles and policies* - The roles and policies used by {hcp-title} can be divided into AWS-managed account-wide roles and policies and Operator roles and policies.
++
+The policies determine the allowed actions for each of the roles. See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS] for more details about the individual roles and policies.
++
+** The AWS-managed account-wide roles are:
++
+*** ManagedOpenShift-Installer-Role
+*** ManagedOpenShift-Worker-Role
+*** ManagedOpenShift-Support-Role
++
+** The AWS-managed account-wide policies are:
++
+*** ManagedOpenShift-Installer-Role-Policy
+*** ManagedOpenShift-Worker-Role-Policy
+*** ManagedOpenShift-Support-Role-Policy
+*** ManagedOpenShift-openshift-ingress-operator-cloud-credentials ^[1]^
+*** ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent ^[1]^
+*** ManagedOpenShift-openshift-cloud-network-config-controller-cloud ^[1]^
+*** ManagedOpenShift-openshift-machine-api-aws-cloud-credentials ^[1]^
+*** ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede ^[1]^
+*** ManagedOpenShift-openshift-image-registry-installer-cloud-creden ^[1]^
++
+[.small]
+--
+1. This policy is used by the cluster Operator roles, listed below. The Operator roles are created in a second step because they are dependent on an existing cluster name and cannot be created at the same time as the AWS-managed account-wide roles. 
+--
++
+** The Operator roles are:
++
+*** <cluster-name\>-xxxx-openshift-cluster-csi-drivers-ebs-cloud-credent
+*** <cluster-name\>-xxxx-openshift-cloud-network-config-controller-cloud
+*** <cluster-name\>-xxxx-openshift-machine-api-aws-cloud-credentials
+*** <cluster-name\>-xxxx-openshift-cloud-credential-operator-cloud-crede
+*** <cluster-name\>-xxxx-openshift-image-registry-installer-cloud-creden
+*** <cluster-name\>-xxxx-openshift-ingress-operator-cloud-credentials
```

Comment:
As rosacli supports to create operator-roles prior cluster spec, which can specify operator-roles prefix, it'd better use <operator-prefix>-****** rather than "<cluster-name\>-xxxx-" for the operator-roles prefix.

Reply:
Would something like `<operator_role_prefix>-openshift-cluster-csi-drivers-ebs-cloud-credentials` work? I would like to keep some words in the < > as an explanation.